Modules/CodeTrek/Resources/views/modals/sidebar-feedback-modal.blade.php

```diff
@@ -0,0 +1,86 @@
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="card-body">
+                        <div class="d-flex justify-content-center justify-content-around">
+                            <div class="form-group col-md-6 text-break">
+                                <h6 class="fw-bold" for="first_name">Name</h6>
+                                <h5>{{ $codeTrekApplicant->first_name }} {{ $codeTrekApplicant->last_name }}</h5>
+                                <input type="hidden" name="latest_round_name" id="latest_round_name" value="{{ $codeTrekApplicant->latest_round_name }}">
+                                <input type="hidden" name="candidate_id" id="candidate_id" value="{{ $codeTrekApplicant->id }}">
+                            </div>
+                            <div class="form-group col-md-6 text-break">
+                                <h6 for="email">Email</h6>
+                                <h5>{{ $codeTrekApplicant->email }}</h5>
+                            </div>
+                        </div>
+                        <div class="d-flex justify-content-center justify-content-between">
+                            <div class="form-group col-md-6 text-break">
+                                <h6 for="university">University</h6>
+                                <h5>{{ $codeTrekApplicant->university }}</h5>
+                            </div>
+                            <div class="form-group col-md-6 text-break">
+                                <h6 for="graduation_year">Graduation Year</h6>
+                                <h5>{{ $codeTrekApplicant->graduation_year }}</h5>
+                            </div>
+                        </div>
+                    <hr>
+                    <div class="d-flex justify-content-start">
+                        <div class="form-group col-lg-6 col-md-5">
+                            <div class="mb-2">
+                                <label for="feedback_categories">Feedback Categories</label>
+                            </div>
+                            <select name="feedback_category" id="feedback_category" class="form-control" required>
+                                <option value="">{{ __('Select Category') }}</option>
+                                @foreach ($feedback_categories as $feedback_category)
+                                    <option value="{{ $feedback_category->id }}">{{ $feedback_category->category_name }}</option>
+                                @endforeach
+                            </select>
+                        </div>
+                        <div class="justify-content-center">
+                            <div class="mb-1 text-center">
+                                <label for="feedback_type">Feedback Type</label>
```

Comment:
Whenever adding a normal text use text sanitizing. You can read more about it in Laravel documentation
```suggestion
                                <label for="feedback_type">{{ __('Feedback Type') }}</label>
```